src/handshake/certificate_verify.rs

```diff
@@ -25,3 +27,18 @@ impl<'a> CertificateVerify<'a> {
         })
     }
 }
+
+#[derive(Debug)]
+#[cfg_attr(feature = "defmt", derive(defmt::Format))]
+pub struct CertificateVerify {
+    pub(crate) signature_scheme: SignatureScheme,
+    pub(crate) signature: heapless::Vec<u8, 128>,
```

Comment:
What is the actual max length of the `ASN.1 der` encoded signature here?

Reply:
That depends the cipher, should perhaps be configurable, but 128 _should_ be sufficient I think.

Reply:
I'll leave it as is for now.